### PR TITLE
feat: make chain-integration > cosmos-sdk links clickable

### LINKED ISSUE #864: integrations links to cosmos-sdk v0.45 docs: Page Not Found
The links we recently aded from https://docs.agoric.com/guides/integration/chain-integration.html to https://docs.cosmos.network/v0.45/ no longer work: **Page Not Found**.

I reported it upstream by way of [a post in the comsos discord dev-forum](https://discord.com/channels/669268347736686612/1158113303114031184/1158113303114031184).

I wonder if https://github.com/cosmos/cosmos-sdk/issues would be a better place to report it.

cc @dtribble @michaelfig @gibson042 @ivanlei 

see also:

 - https://github.com/Agoric/documentation/issues/657

### PR DESCRIPTION
**Pull request by 0xpatrickdev**
- (sort of) closes #864, which claims these links do not work

## Before
image

## After 
image

### CODE REVIEW

File: main/guides/integration/chain-integration.md [hunk 1]
```diff
@@ -36,15 +36,15 @@ agd version --long
 # FAQ
 
 - How are transactions encoded?
-https://docs.cosmos.network/v0.45/core/encoding.html
+[https://docs.cosmos.network/v0.45/core/encoding.html](https://docs.cosmos.network/v0.45/core/encoding.html)
```
Comment (0xpatrickdev) [inline main/guides/integration/chain-integration.md:39]: These links also work with `v0.46`, but 404 at `v0.47` and `main`

File: main/guides/integration/chain-integration.md [hunk 2]
```diff
@@ -36,15 +36,15 @@ agd version --long
...
 - What data is needed to create a transaction (last block hash, nonce, sender public key, etc.)?
 https://docs.cosmos.network/v0.45/core/transactions.html#transaction-generation
 - What data is signed and how is that data obtained (for example truncated SHA256 of transaction data)?
-https://docs.cosmos.network/v0.45/core/transactions.html#transaction-generation
+[https://docs.cosmos.network/v0.45/core/transactions.html#transaction-generation](https://docs.cosmos.network/v0.45/core/transactions.html#transaction-generation)
```
Comment (dckc) [inline main/guides/integration/chain-integration.md:43]: Let's avoid urls-in-your-face, please:

```suggestion
[Transaction Generation](https://docs.cosmos.network/v0.45/core/transactions.html#transaction-generation)
```

and likewise throughout.

**Review by dckc — COMMENTED** [review]
I'd rather we used normal link text

File: main/guides/integration/chain-integration.md (hunk 2 above)
Comment (0xpatrickdev) [reply main/guides/integration/chain-integration.md]: [eb9064d](https://github.com/Agoric/documentation/pull/970/commits/eb9064dd85a1bf658e75a44a76b6d336335d30f3) (updated PR body with new After)

**Review by dckc — APPROVED** [review]
thanks for tidying up around the shop

**Review by kbennett2000 — APPROVED** [review]
Looks much better!

**Pull request merged**